spec/system/kudo_system_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  # rubocop:disable RSpec/ExampleLength
+
+  it 'enables me to create, edit and destroy kudos' do
+    FactoryBot.create :employee,  email: 'test3@test.pl',
```

Comment:
you included FactoryBot syntax methods, so you should be able to write simply - when only name of the factory,
also please check out let() syntax

```suggestion
    employee = create(:employee)
```

Reply:
All suggestions corrected. Thanks!